semgrep.opam

```diff
@@ -41,7 +41,7 @@ depends: [
   "easy_logging_yojson" { = "0.8.1" }
   "logs"
   "atdgen"
-  "yojson"
+  "yojson" { >= "2.0.0" }
```

Comment:
we might want to update ocaml-layer too
(remove comby-kernel in there too).

Reply:
yes, I did that too :) https://github.com/returntocorp/ocaml-layer/pull/20